src/meta/processors/configMan/SetConfigProcessor.cpp

```diff
@@ -21,43 +21,37 @@ void SetConfigProcessor::process(const cpp2::SetConfigReq& req) {
     if (module != cpp2::ConfigModule::ALL) {
         // When we set config of a specified module, check if it exists.
         // If it exists and is mutable, update it.
-        if (!setOneConfig(module, name, type, mode, value, data)) {
```

Comment:
Why did we remove the result check here?

Reply:
If `setOneConfig` return false, which means the config doesn't exist or it is immutable. We just return error at below.